_posts/0000-01-02-Electroman856.md

```diff
@@ -0,0 +1,4 @@
+layout: slide
+title: "Welcome to our second slide!"
+
+
```

Comment:
<H1>
<p>  Hello everybody. Im trying some code that I am sure is not going to work. </P>